src/profile-logic/process-profile.js

```diff
@@ -223,26 +248,117 @@ function processThread(thread, libs) {
             );
           }
           fileName = stringTable.indexForString(scriptURI);
-          lineNumber = jsMatch[3] | 0;
+          lineNumber = parseInt(jsMatch[3], 10);
```

Comment:
Flow couldn't figure the original out, I couldn't either without running it, lol.